src/System.Management.Automation/engine/parser/SemanticChecks.cs

```diff
@@ -458,13 +458,12 @@ public override AstVisitAction VisitSwitchStatement(SwitchStatementAst switchSta
             // Parallel flag not allowed
```

Comment:
@KirkMunro, your last commit had 4 failures in `PowerShell-CI-linux`
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Language/Scripting/ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```